lib/rubocop/cop/style/one_line_conditional.rb

```diff
@@ -25,11 +25,13 @@ module Style
       class OneLineConditional < Cop
         include OnNormalIfUnless
 
-        MSG = 'Favor the ternary operator (`?:`) ' \
-              'over `%<keyword>s/then/else/end` constructs.'
+        MSG_USE_TERNARY = 'Favor the ternary operator (`?:`) ' \
+                          'over `%<keyword>s/then/else/end` constructs.'
+        MSG_USE_MULTILINE = 'Use multiple lines for ' \
```

Comment:
Seems to me that this change is more than a bugfix - it actually changes the behaviour of the cop. Why would we want to treat this case differently?